scripts/convert-bson-corpus-tests.php

```diff
@@ -238,6 +238,7 @@ function getParamsForParseError(array $test, array $case)
 
     switch ($test['bson_type']) {
         case '0x00': // Top-level document
+        case '0x05': // Binary
```

Comment:
This reminded me of [DRIVERS-1552](https://jira.mongodb.org/browse/DRIVERS-1552). I'll trust this is correct, since that issue hasn't been discussed since I reported it.